components/NavigationBar/examples/NavigationBarLayoutExample.m

```diff
@@ -1,24 +1,19 @@
 /*
  Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
+
  http://www.apache.org/licenses/LICENSE-2.0
+
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
  */
 
-//
-//  NavigationBarLayoutExample.m
-//  Pods
-//
-//  Created by Rob Moore on 7/28/17.
-//
-//
-
```

Comment:
I appreciate this cleanup, but changes to license clauses can draw extra scrutiny.
Consider pulling the changes to this file into a separate PR.